src/content/configuration/cli.mdx

```diff
@@ -202,13 +202,18 @@ Some options can be configured through environment variables. You will typically
 | `6`       | `BUILD_WAS_CANCELED`         | Chromatic build was canceled                           |
 | `11`      | `ACCOUNT_QUOTA_REACHED`      | Chromatic account reached its snapshot quota           |
 | `12`      | `ACCOUNT_PAYMENT_REQUIRED`   | Chromatic account requires payment                     |
+| `21`      | `STORYBOOK_BUILD_FAILED`     | Failed to run build-storybook script                   |
```

Comment:
@jonniebigodes could you confirm that these descriptions are correct

Reply:
Yea, but I want to explain what each error means saying "Storybook issue" is not very helpful for the user

Reply:
I agree with @winkerVSbecks here, and we should be a bit more explicit. The `STORYBOOK_BUILD_FAILED` most common cause is that the script is not found or something of that nature. The `STORYBOOK_BROKEN` seems to be a catch-all event to be used when there's a certain build failure (see [here](https://github.com/chromaui/chromatic-cli/blob/main/node-src/tasks/verify.ts#L199) for the usage), so naming it load might be a bit off. Regarding the `STORYBOOK_START_FAILED`, I only see it in the exit codes definition file and not used anywhere. I might be wrong, but I suspect that this is something that comes from the platform to notify users of the build URL provided by Chromatic.

Reply:
Thanks @jonniebigodes. Yea, I couldn't find where `STORYBOOK_START_FAILED` is used either